content/c/concepts/structures/structures.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: 'Structures'
+Description: 'A structure can be used to group different types of data together.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Lists'
+  - 'Variable Types'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A structure can be used to group different types of data together. It is defined using the `struct` keyword.
+
+## Syntax
+
+```pseudo
+struct Name{
+  type1 memberName1;
+  type2 memberName2;
+  type3 memberName3;
+  ...
+  typeN memberNameN;
+}
+```
+
+A structure is made up of members where each `memberName` has to be declared with a `type`.
+
+## Example
+
+The following example uses the `struct` keyword to create a `Patient` structure:
+
+```c
+struct Patient{
+  char name[30];
+  int age;
+  int weight;
+  int heightInInches;
+};
+```
+
+`name`, `age`, `weight`, and `heightInInches` are members of the `Patient` structure.
+
+## Structure Variables
+
+[Variables](https://www.codecademy.com/resources/docs/c/variables) can be created in order to work with the `Patient` structure above. The following example declares and initializes a structure variable:
+
+```c
+#include <stdio.h>
+#include <string.h>
+
+struct Patient{
+  char name[30];
+  int age;
+  int weight;
+  int heightInInches;
+};
+
+int main() {
+
+  // Variable initialization at declaration
+  struct Patient patientA = {"Douglas Franklin", 62, 280, 74};
+  return 0;
+
+}
+```
+
+## Accessing Members
+
+Members can be accessed using the member operator `.`:
```

Comment:
This is more of a thought than a suggestion, but should we eventually edit the [C: Operators](https://www.codecademy.com/resources/docs/c/operators) entry by including info about the member operator?

Reply:
@Dusch4593 Yeah, we should. I think we should open an issue on it.

Reply:
Actually, now that I think about it, I'm not sure if the member operator would fit in there. The operators perform a specific mathematical or logical operation and the member operator doesn't exactly do that 🤔